media-fonts/inconsolata-lgc/inconsolata-lgc-1.3.0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit font
```

Comment:
We usually leave one empty line below EAPI to follow skel format.

Reply:
Sure, blank line added.